task_geo/data_sources/mobility/audit.md

```diff
@@ -0,0 +1,53 @@
+# GOOGLE MOBILITY REPORTS
+
+## General information
+
+- **Description**: Contains the variations in mobility tracked by Google worldwide.
+    The data is presented at both country and region level, with lesser granularity in some cases (US).
+- **Credits**: Google.
+- **Source**: https://www.google.com/covid19/mobility/
+
+## Description
+
+**country_iso**
+- Description: ISO-2 code for the country
+- Type: string
+
+
+**region**
+- Description: Name of the region
+- Type: string
+
+**date**
+- Description: The date the measurements corresponds to
+- Type: date.
+
+**retail_recreation**
+- Description: Variance in the given category in relation to the baseline.
```

Comment:
```suggestion
- Description: Changes in movement for places classified as retail and recreation in relation to the baseline.
```
(same for the other categories-- it's a bit easier to know what's going on without going to the google mobility site)